Code/Editor/ViewportManipulatorController.cpp

```diff
@@ -114,16 +114,24 @@ namespace SandboxEditor
             // Cache the ray trace results when doing manipulator interaction checks, no need to recalculate after
             if (event.m_priority == ManipulatorPriority)
             {
+
```

Comment:
nit: remove extra linebreak

Reply:
To have this work exclusively with manipulators too, I think you need to look at the return from...

```
InteractionBus::EventResult(
    interactionHandled, AzToolsFramework::GetEntityContextId(), targetInteractionEvent, mouseInteractionEvent);
```

You only want to do this when dragging manipulators, see this line...

```
const auto& targetInteractionEvent = event.m_priority == ManipulatorPriority
    ? &InteractionBus::Events::InternalHandleMouseManipulatorInteraction
     : &InteractionBus::Events::InternalHandleMouseViewportInteraction;
```

so we only care when the priority is manipulators and the return is 'consumed' I believe in on mouse down. The `interactionHandled` variable. Let me know if that's any help or if you have more questions 👍 

Great work so far! 😄